modules/tensor_mechanics/src/materials/ComputeMultipleInelasticCosseratStress.C

```diff
@@ -42,6 +42,9 @@ ComputeMultipleInelasticCosseratStress::initQpStatefulProperties()
 void
 ComputeMultipleInelasticCosseratStress::computeQpStress()
 {
+  if (!_elasticity_tensor_is_constant[_qp])
+    mooseError("The Cosserat stress classes assume constant elasticity tensors");
+
```

Comment:
Looks fine.  Perhaps when things "shake down" a little in this PR this Cosserat-constant check might be (re)moved, i'm not sure.